template/7/function/serverless.yml

```diff
@@ -17,5 +17,5 @@ functions:
 
 # Exclude files from deployment
 package:
-    exclude:
-        - 'tests/**'
+    pattern:
```

Comment:
I was looking into this change and it appears that `pattern` is an invalid key. The correct key would be `patterns`, perhaps?

```
Serverless:   at 'package': unrecognized property 'pattern'
```

Reply:
Yes indeed 🤦
I'll look into this soon™️

Didn't had the time to work on this again and test it